distribution/environments/aws-ec2-cloud/config/as-code/artifact-manager-s3.yaml

```diff
@@ -1,14 +1,14 @@
 ---
-# TODO: file JIRAs to make this classifed (and for other used plugins too)
 unclassified:
-  s3blobstoreconfig:
-    # TODO: create through CloudFormation
-    container: "${ARTIFACT_MANAGER_S3_BUCKET_NAME}"
+  artifactManager:
+    artifactManagerFactories:
+      - jclouds:
+          provider: s3
+aws:
+  awsCredentials:
```

Comment:
Note that while https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/72 tests this format, I did not attempt to test this actual file patch, since I have no idea how to do so—acc. to discussion with @batmat there appears to be no equivalent to

```sh
make ENV=aws try-everything
```

which would use (say) Docker Compose to start up a local copy of the backend and the client so you can go to `http://localhost:8080/` and poke around.

Also it seems that (acc. to [JENKINS-52581](https://issues.jenkins-ci.org/browse/JENKINS-52581)) `Jenkinsfile` currently runs no tests which would even verify that this file applies without error.

Reply:
So, I was unclear I think. We do not have a way currently to test the specific image built for AWS, that is indeed the gist of [JENKINS-52581](https://issues.jenkins-ci.org/browse/JENKINS-52581). But by running either `make check` from the repo root, or the more specific `make container-check` under the `distribution` directory, you would already run a bunch of tests. Seems one is already failing, see https://github.com/jenkins-infra/evergreen/pull/144#issuecomment-405567940 below.

Reply:
Build running in https://github.com/jenkins-infra/evergreen/pull/146 @jglick if this succeeds, as I think it should, you should be able to use it locally to check at least the server starts for instance (which is not the case currently).

Reply:
Well when that is merged, then I can see whether deliberately bogus plugin definitions and/or YAML configuration indeed result in failures.